packages/cra-template-imodeljs/template.json

```diff
@@ -0,0 +1,48 @@
+{
+  "package": {
+    "dependencies": {
+      "@bentley/bentleyjs-core": "^2.0.0",
+      "@bentley/extension-client": "^2.0.0",
+      "@bentley/frontend-authorization-client": "^2.0.0",
+      "@bentley/geometry-core": "^2.0.0",
+      "@bentley/imodelhub-client": "^2.0.0",
```

Comment:
if we use `^` and dont provide a lock file it'll pull in the latest which is 2.4 right? and if so it'll complain about the new rbac-client dep, so maybe that should be added here as well. and we use ^2.4